lms/security.py

```diff
@@ -101,6 +142,37 @@ def forget(self, request):
         pass
 
 
+class LTILaunchesSecurityPolicy(LTIUserSecurityPolicy):
+    """
+    Security policy for LTI launches
```

Comment:
This security policy doesn't use `get_lti_user` and instead just looks at the schemas relevant for LTI authentication (as in LTI spec, not as in our LTIUser) and handles the exceptions.